tests/python/rest_api/test_projects.py

```diff
@@ -568,44 +568,29 @@ def test_can_import_dataset_in_org(self, admin_user):
 
         self._test_import_project(admin_user, project_id, "CVAT 1.1", import_data)
 
-    def test_can_export_and_import_dataset_with_skeletons_coco_keypoints(self, admin_user):
-        project_id = 5
-
-        response = self._test_export_project(admin_user, project_id, format="COCO Keypoints 1.0")
-
-        tmp_file = io.BytesIO(response.data)
-        tmp_file.name = "dataset.zip"
-        import_data = {
-            "dataset_file": tmp_file,
-        }
-
-        self._test_import_project(admin_user, project_id, "COCO Keypoints 1.0", import_data)
-
-    def test_can_export_and_import_dataset_with_skeletons_cvat_for_images(self, admin_user):
-        project_id = 5
-
-        response = self._test_export_project(admin_user, project_id)
-
-        tmp_file = io.BytesIO(response.data)
-        tmp_file.name = "dataset.zip"
-        import_data = {
-            "dataset_file": tmp_file,
-        }
-
-        self._test_import_project(admin_user, project_id, "CVAT 1.1", import_data)
-
-    def test_can_export_and_import_dataset_with_skeletons_cvat_for_video(self, admin_user):
+    @pytest.mark.parametrize(
+        "export_format, import_format",
+        (
+            ("COCO Keypoints 1.0", "COCO Keypoints 1.0"),
+            ("CVAT for images 1.1", "CVAT 1.1"),
```

Comment:
I have some doubts here: previously the test for CVAT for images relied on default format. Should I keep this behaviour?

Reply:
It's ok, the CVAT formats are supposed to be default. I think it's better now. One thing here that can be improved is that the source annotations can be checked to contain skeletons. It's either through the `annotations` fixture, or the explicit request to `/api/(projects|tasks|jobs)/{id}/annotations`.

Reply:
Now the test searches for project with skeleton annotations